host-knc.config

```diff
@@ -14,7 +14,9 @@
       "tag/compiler/gcc",
       "unordered_map",
       "fstream",
-      ] 
+      "sys/types.h",
```

Comment:
the header files provided by the system can go into kernel/build/gcc-host-pc/mcconf.module